app/src/main/java/com/framgia/moneymanagement/screen/statistics/StatisticsAdapter.java

```diff
@@ -0,0 +1,51 @@
+package com.framgia.moneymanagement.screen.statistics;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import com.framgia.moneymanagement.R;
+
+public class StatisticsAdapter extends FragmentPagerAdapter {
+    private Context mContext;
+    private static final int NUMBER_ZEZO = 0;
```

Comment:
ko ai define constant ntn cả. 
E cần define theo vai trò của nó chứ k phải giá trị của nó
```suggestion
    private static final int STATISTICS_LAST_MONTH_POSITION = 0;
```